<ui version="4.0" >
 <class>Dialog_options</class>
 <widget class="QDialog" name="Dialog_options" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="use_flip_checkbox" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>20</y>
     <width>159</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>use flip</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>188</width>
     <height>304</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="verbose_label" >
        <property name="text" >
         <string>Verbose</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="verbose_spinbox" >
        <property name="maximum" >
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="mchoice_label" >
        <property name="text" >
         <string>Multiply Choice #</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="mchoice_spinbox" >
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="percent_label" >
        <property name="text" >
         <string>Init with %</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="percent_spinbox" >
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="norm_tol_label" >
        <property name="text" >
         <string>Normal Tol</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="norm_tol_spinbox" >
        <property name="decimals" >
         <number>3</number>
        </property>
        <property name="maximum" >
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.050000000000000</double>
        </property>
        <property name="value" >
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="tang_tol_label" >
        <property name="text" >
         <string>Tangential Tol</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="tang_tol_spinbox" >
        <property name="decimals" >
         <number>3</number>
        </property>
        <property name="maximum" >
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.050000000000000</double>
        </property>
        <property name="value" >
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="alpha_label" >
        <property name="text" >
         <string>Alpha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="alpha_spinbox" >
        <property name="decimals" >
         <number>3</number>
        </property>
        <property name="maximum" >
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.050000000000000</double>
        </property>
        <property name="value" >
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="relocation_label" >
        <property name="text" >
         <string>Relocation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="relocation_spinbox" />
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="ghost_vs_solid_label" >
        <property name="text" >
         <string>Ghost vs solid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ghost_spinbox" >
        <property name="decimals" >
         <number>3</number>
        </property>
        <property name="maximum" >
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.050000000000000</double>
        </property>
        <property name="value" >
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>210</y>
     <width>174</width>
     <height>109</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="thickness" >
        <property name="text" >
         <string>Line thickness</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="thickness_spinbox" />
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="point_size" >
        <property name="text" >
         <string>Point size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="point_size_spinbox" />
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="vertex_size" >
        <property name="text" >
         <string>Vertex size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="vertex_size_spinbox" />
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
